vaccine_feed_ingest/stages/ingest.py

```diff
@@ -324,17 +324,49 @@ def _validate_normalized(output_dir: pathlib.Path) -> bool:
                     return False
 
                 if normalized_location.location:
-                    if not BOUNDING_BOX.latitude.contains(
-                        normalized_location.location.latitude
-                    ) or not BOUNDING_BOX.longitude.contains(
-                        normalized_location.location.longitude
-                    ):
+                    result, failed_bound = validate_bounding_boxes(
+                        normalized_location.location, [BOUNDING_BOX, BOUNDING_BOX_GUAM]
+                    )
+
+                    # if false, return false
+                    if not result:
                         logger.warning(
                             "Invalid latitude or longitude in %s at line %d: %s is outside approved bounds (%s)",
                             filepath,
                             line_no,
                             normalized_location.location,
-                            BOUNDING_BOX,
+                            failed_bound if failed_bound is not None else "all",
                         )
                         return False
+
     return True
+
+
+def validate_bounding_boxes(location, bounding_boxes, method="all"):
+    results = []
+
+    for boundingbox in bounding_boxes:
+        if not boundingbox.latitude.contains(
+            location.latitude
+        ) or not boundingbox.longitude.contains(location.longitude):
+            if method == "any":
+                # fail if any single bounding box fails
+                return False, boundingbox
+            else:
+                results.append(False)
+
+        results.append(True)
+
+    if method == "all":
```

Comment:
This extra flexibility of methods is cool, but also unnecessary. Since we don't need the flexibility I don't want to add code to maintain that we don't need until we need it.

Can you remove `method` and just check to make sure it is in one of the approved bounding boxes, and if it is then return True.